aiida/orm/implementation/sqlalchemy/group.py

```diff
@@ -128,7 +128,7 @@ def store(self):
 
         return self
 
-    def add_nodes(self, nodes):
+    def add_nodes(self, nodes, skip_orm=False):
```

Comment:
here it would make sense to add a docstring

(the meaning of skip_orm is probably not entirely self-evident)